apps/studio/src/pages/_app.tsx

```diff
@@ -33,6 +33,10 @@ const MyApp = ((props: AppPropsWithAuthAndLayout) => {
             <ErrorBoundary FallbackComponent={DefaultFallback}>
               <Suspense fallback={<Skeleton width="100vw" height="100vh" />}>
                 <Stack spacing={0} minH="$100vh">
+                  <Banner variant="warn">
+                    Isomer Next is currently in Beta. To manage site settings
+                    that are not displayed here, contact Isomer Support.
```

Comment:
since we have growthbook liow. I think we can take this opportunity to wire up the banner text to growthbook. Can see [ActiveSG implementation](https://github.com/opengovsg/activesg/blob/main/packages/ui/src/components/AppBanner.tsx#L36)

Reply:
sounds good, lemme add it in